modules/ROOT/pages/sync-function-api.adoc

```diff
@@ -368,10 +368,14 @@ When running a re-sync operation, the context in the Sync Function is the admin
 For that reason, calling the `requireUser`, `requireAccess` and `requireRole` methods will always succeed.
 It is very likely that you are using those functions in production to govern write operations.
 But in a re-sync operation, all the documents are already written to the database.
-For that reason, it is recommended to use re-sync for changing the assignment to channels only (i.e.
-reads). If the modifications to the Sync Function only impact write security (and not routing/access), you won't need to run the re-sync operation.
+For that reason, it is recommended to use re-sync for changing the assignment to channels only (i.e. reads).
 
-Similarly, if you wish to change the channel/access rules, but only want those rules to apply to documents written after the change was made, then you don't need to run the re-sync operation.
+If the modifications to the Sync Function only impact write security (and not routing/access), you won't need to run the re-sync operation.
+
+If you wish to change the channel/access rules, but only want those rules to apply to documents written after the change was made, then you don't need to run the re-sync operation.
+
+If you change the sync function to revoke a user's access to a document, the access will only take affect once a new revision to that document is saved on Sync Gateway.
+Running a re-sync operation does not revoke access to that document immediately.
```

Comment:
I'd get rid of the word "immediately" in this sentence, re-sync won't revoke that document ever. Just to make it super clear that a new revision of the doc is required.